dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/openapi/ApiAnalyse.java

```diff
@@ -337,7 +337,7 @@ else if ( p.isAnnotationPresent( RequestBody.class ) )
                 RequestBody a = p.getAnnotation( RequestBody.class );
                 Api.RequestBody requestBody = endpoint.getRequestBody()
                     .init( () -> new Api.RequestBody( p, a.required() ) );
-                Api.Schema type = analyseParamSchema( endpoint, p.getParameterizedType(), p.getType() );
+                Api.Schema type = analyseParamSchema( endpoint, p.getParameterizedType() );
```

Comment:
FYI: This is a bugfix: when the `p.getType()` was passed this was causing request bodies with parameterized types to not be fully mapped correctly.